DisplaySettings/DisplaySettings.cpp

```diff
@@ -314,12 +315,18 @@ namespace WPEFramework {
 			catch (const device::Exception& err){
 				LOG_DEVICE_EXCEPTION1(string("HDMI_ARC0"));
 			}
+			try {
+		    		m_hdmiCecAudioDeviceDetected = getHdmiCecSinkAudioDeviceStatus();
```

Comment:
AudioDeviceConnectedStatus so rename API and variable accordingly

Reply:
Will change the API name to "getHdmiCecSinkAudioDeviceConnectedStatus" and let the variable name "m_hdmiCecAudioDeviceDetected" remain same, because there is an other variable of m_hdmiInAudioDeviceConnected which would create confusions. Anyways the m_hdmiCecAudioDeviceDetected variable indicate the audio device detected.

Reply:
m_hdmiCecAudioDeviceDetected make sense... I gave generic comment address where where applicable